src/components/Icon/Icon.tsx

```diff
@@ -1,11 +1,83 @@
-interface IconProps {
-  svg: string;
-  iconSize: number;
-  onClick: () => void;
+import styles from './icon.module.css';
+
+interface IconCartProps {
+  itemCount: number;
 }
 
-const Icon = (_: IconProps) => {
-  return <></>;
-};
+export const CartIcon = ({ itemCount }: IconCartProps) => (
+  <div className={styles.container}>
+    <svg width="24" height="24" viewBox="0 0 24 24" fill="none">
+      <path
+        d="M6.12403 9.03885C6.2568 7.97675 7.42732 7.39149 8.35666 7.92254L10.7597 9.29568C11.5282 9.73487 12.4718 9.73487 13.2403 9.29568L15.6433 7.92254C16.5727 7.39149 17.7432 7.97675 17.876 9.03885L19.2229 19.8139C19.3348 20.7092 18.6367 21.5 17.7344 21.5H6.26556C5.36332 21.5 4.66524 20.7092 4.77715 19.8139L6.12403 9.03885Z"
+        stroke="#0D1222"
```

Comment:
help me research about `currentColor` in svg, that will help change color of the icon depend of the parent element

Reply:
Fixed